### PR TITLE
ENH get rid of the rest of the xonsh

### PR DESCRIPTION
**Pull request by beckermr**
This PR gets rid of the rest of the xonsh.

### CODE REVIEW

File: conda_forge_tick/audit.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 from conda_forge_tick.contexts import MigratorSessionContext, FeedstockContext
 from conda_forge_tick.git_utils import feedstock_url
-from conda_forge_tick.git_xonsh_utils import fetch_repo
+from conda_forge_tick.git_utils import fetch_repo
```
Comment (CJ-Wright) [inline conda_forge_tick/audit.py:19]: Can you merge this import with the one above?

File: conda_forge_tick/cli.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+
+def deploy(args):
+    """Deploy the graph to github"""
+    if args.dry_run:
+        print("(dry run) deploying")
+        return
+
+    CIRCLE_BUILD_URL = os.environ.get("CIRCLE_BUILD_URL", "")
+    for cmd in [
```
Comment (CJ-Wright) [inline conda_forge_tick/cli.py:42]: might as well collapse a bunch of the `git add`s with a list comp?

**Review by CJ-Wright — COMMENTED** [review]
just some light refactors

File: conda_forge_tick/audit.py (hunk 1 above)
Comment (beckermr) [reply conda_forge_tick/audit.py:19]: What do you mean? I changed the imports everywhere in the code base.

File: conda_forge_tick/cli.py (hunk 2 above)
Comment (beckermr) [reply conda_forge_tick/cli.py]: IDK if the code is clearer now or not but here you go!

File: conda_forge_tick/audit.py (hunk 1 above)
Comment (beckermr) [reply conda_forge_tick/audit.py:19]: 🤦  hold on

Comment (beckermr) [reply conda_forge_tick/audit.py]: done.

**Review by CJ-Wright — APPROVED** [review]
Thank you very much!

**Pull request merged**